tests/conftest.py

```
@@ -57,22 +63,88 @@ def pytest_addoption(parser):
+    Computes the Sun at probe dates chosen inside one tier's range but outside
+    the next narrower one: year 1000 is covered only by DE441 (extended),
+    year 1700 by DE440 (medium) but not DE440s (base, 1849+).
+    """
+    from kerykeion.ephemeris_backend import swe, ephemeris_session
+
+    def _sun_computes(year: int) -> bool:
+        try:
+            with ephemeris_session() as iflag:
+                swe.calc_ut(swe.julday(year, 1, 1, 12.0), 0, iflag)
+            return True
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)